src/Sensor/BmsFault.h

```diff
@@ -4,93 +4,78 @@
-        THERMISTOR_PACK,
-        THERMISTOR_INTERNAL, 
+        CELL_OVER_5V,
+        VOLTAGE_HIGH, 
         CURRENT_DISCHARGE_HIGH,
         CURRENT_CHARGE_HIGH,
-        CURRENT_REGEN_HIGH,
-        CURRENT_SENSOR_DISCONNECT,
-        CURRENT_SENSOR_CONNECT,
         SWITCH_CHARGE,
         SWITCH_DISCHARGE,
-        SOFTWARE,
+        OPEN_WIRING,
+        CURRENT_SENSOR_DISCONNECT,
         HARDWARE,
         INTERNAL_LOGIC,
         COMMUNICATION_INTERNAL,
+        CELL_BANK,    
+        BALANCING,
+        THERMISTOR_PACK,
+        THERMISTOR_INTERNAL, 
         COMMUNICATION_CAN,
-        OPEN_WIRING,
-        WEAK_CELL,
+        VOLTAGE_LOW,
+        TEMP_LOW,
         WEAK_PACK,
-        BALANCING,
-        POWER_SUPPLY,
-        HIGH_VOLTAGE_ISOLATION,
-        CELL_BANK    
+        WEAK_CELL,
+        CURRENT_SENSOR_CONNECT
     };
 
     inline String toString(uint8_t code) {
         switch(code) {
             case NONE:
                 return "None";
-            case VOLTAGE_LOW:
-                return "Low Voltage";
-            case VOLTAGE_HIGH:
-                return "High Voltage";
-            case CELL_OVER_5V:
-                return "Cell Voltage >5v";
-            case TEMP_LOW:
-                return "Low Temperature";
             case TEMP_HIGH:
                 return "High Temperature";
-            case TEMP_BMS_HIGH:
-                return "High BMS Temperature";
-            case THERMISTOR_PACK:
-                return "Battery Pack Thermistor";
-            case THERMISTOR_INTERNAL:
-                return "BMS Thermistor";
+            case CELL_OVER_5V:
+                return "Cell Voltage >5v";
+            case VOLTAGE_HIGH:
+                return "High Voltage";
             case CURRENT_DISCHARGE_HIGH:
                 return "High Discharge Current";
             case CURRENT_CHARGE_HIGH:
                 return "High Charge Current";
-            case CURRENT_REGEN_HIGH:
-                return "High Regen Current";
-            case CURRENT_SENSOR_DISCONNECT:
-                return "Current Sensor Disconnected";
-            case CURRENT_SENSOR_CONNECT:
-                return "Current Sensor Connected";
             case SWITCH_CHARGE:
                 return "Charge Switch";
             case SWITCH_DISCHARGE:
                 return "Discharge Switch";
-            case SOFTWARE:
-                return "Software";
+            case CURRENT_SENSOR_DISCONNECT:
+                return "Current Sensor Disconnected";
             case HARDWARE:
                 return "Hardware";
             case INTERNAL_LOGIC:
                 return "Internal Logic";
             case COMMUNICATION_INTERNAL:
                 return "Internal Communication";
-            case COMMUNICATION_CAN:
-                return "CAN Communication";
+            case CELL_BANK:
+                return "Cell Bank";
+            case BALANCING:
+                return "Balancing";
             case OPEN_WIRING:
```

Comment:
@valentino-jaber noticed this in the develop_bms_fault branch, but OPEN_WIRING is out of order in this switch statement. This will make the jump table less efficient.

Reply:
k I'll change that now